src/sdcard.c

```diff
@@ -180,6 +180,7 @@ void mega65_sdcard_unmap_sector_buffer(void)
 
 unsigned short timeout;
 
+// @todo Return -1 corresponds to 255. Is this what we want?
```

Comment:
I assume all bits set is an error value. This corresponds a bit to `EOF` which is used by some C library functions for error result.
Typically such cases use `int` in as the return type. This may be less ideal on an 8-bit CPU when given the choice of return type.
Personally I think it is ugly to return -1 when the return type is unsigned. One can use `~0` or `~0u`, or perhaps better a macro to represent it.
Another way is to use `int8_t` as the return type which would signal that one can test for error by testing if the result is less than 0. In such case a negative value does not look inappropriate, which it kind of does now.